src/main/scala/commonist/util/OaiPmh2.scala

```diff
@@ -0,0 +1,122 @@
+package commonist.util
+
+import commonist.data._
+
+import scala.xml._
+
+import scutil.log._
+
+/** 
+ * Parse and extract image metadata from OAI-PMH 2.0 files.
+ * See www.openarchives.org/OAI/openarchivesprotocol.html
+ */
+class OaiPmh2(doc:Elem,props:Map[String,String]) extends Logging {
+
+  /** fix badly formatted XML files (escaped twice) **/
```

Comment:
i see what this does - but _why_ does it need to be done?

Reply:
The data I got from my city archives contains this kind of tags:

`<dc:format>NEGATIF N&amp;amp;B ; 9 x 12 cm ; Photographie ; PLAQUE DE VERRE</dc:format>`

`<dc:description>Image publiée dans &amp;amp;quot;La Photographie à l'assaut des Pyrénées&amp;amp;quot;, de Santiago Mendieta (Glénat, 2004), p. 9.</dc:description>`